scripts/test/mender-convert-qemu

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+
+set -e
+
+if [ -z "$DISK_IMG" ]; then
+    echo "ERROR: DISK_IMG is not set"
+    exit 1
+fi
+
+# Try common locations for OVMF
+ovmf_file=""
+for maybe_ovmf_file in /usr/share/OVMF/OVMF_CODE.fd /usr/share/ovmf/x64/OVMF_CODE.fd; do
+    if [ -f $maybe_ovmf_file ]; then
+        ovmf_file=$maybe_ovmf_file
+    fi
+done
+
+if [ -z "${ovmf_file}" ]; then
+    echo "ERROR: could not find OVMF file"
+    exit 1
+fi
+
+qemu-system-x86_64 \
+    -enable-kvm \
+    -nographic \
+    -m 256 \
+    -net user,hostfwd=tcp::8822-:22 \
```

Comment:
minor: just for clarity, maybe lets create `MENDER_SSH_PORT=8822` variable?

Reply:
This is hardcoded in the test framework and not possible to change that easily. The meta-mender script does it [the same way](https://github.com/mendersoftware/meta-mender/blob/master/meta-mender-qemu/scripts/mender-qemu#L128).